library/ui/src/main/java/com/google/android/exoplayer2/ui/AspectRatioFrameLayout.java

```diff
@@ -51,6 +51,10 @@
    * The specified aspect ratio is ignored.
    */
   public static final int RESIZE_MODE_FILL = 3;
+  /**
+   * The height or width is increased or decreased to crop and to obtain the desired aspect ratio.
+   */
+  public static final int RESIZE_MODE_CROP = 4;
```

Comment:
I actually wonder about the variable name too. From the point of view of this class, it's not really doing any cropping. It's just making its bounds bigger. Maybe it should be called RESIZE_MODE_EXPAND or something (not that that's a particularly good name for it either...).

Reply:
Yeap, sounds much better than ASPECT_FILL mode :)